Sources/alloy-codeless-lite-ios/Public/Utility/UIUtils.swift

```diff
@@ -10,12 +10,15 @@ import UIKit
 internal struct UIUtils {
 
     @MainActor
-    static func dismissCurrentView() {
-        UIApplication
-            .shared
-            .connectedScenes
-            .compactMap { ($0 as? UIWindowScene)?.keyWindow }
-            .last?.rootViewController?.presentedViewController?.dismiss(animated: true)
+    static func dismissCurrentView() {        
```

Comment:
is it worth being super safe and checking for the version to run this and keep the old one if the version is higher than 13?

looks like you can do it like [this](https://stackoverflow.com/questions/24503001/check-os-version-in-swift)